docs/.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "@tresjs/eslint-config-vue",
+  "rules": {
+    "no-console": "off"
```

Comment:
I would say, no console for docs too.
Where we ignore this rule is in the playground

Reply:
woops. This was meant to go in the playground eslinrc.json. Thanks for noticing!